src/ReferenceTrimmerTask.cs

```diff
@@ -12,22 +10,10 @@ namespace ReferenceTrimmer
 {
     public sealed class ReferenceTrimmerTask : MSBuildTask
     {
-        private static readonly HashSet<string> NugetAssemblies = new(StringComparer.OrdinalIgnoreCase)
```

Comment:
Isn't this still needed for parsing the lock file?

Reply:
In particular I believe this is needed if the version the task depends on is a different version than MSBuild's?